tests/settings/SectionFillingTest.py

```diff
@@ -38,34 +38,37 @@ class SectionFillingTest(unittest.TestCase):
 
     def setUp(self):
         self.log_printer = LogPrinter(ConsolePrinter())
-        self.section = Section("test")
-        self.section.append(Setting("key", "val"))
+        self.section1 = Section("test1")
+        self.section2 = Section("test2")
+        self.section1.append(Setting("key", "val"))
+        self.section2.append(Setting("key", "val"))
 
     def test_fill_settings(self):
-        sections = {"test": self.section}
+        sections = {"test1": self.section1, "test2": self.section2}
         with simulate_console_inputs() as generator:
             fill_settings(sections,
                           acquire_settings,
                           self.log_printer)
             self.assertEqual(generator.last_input, -1)
 
-        self.section.append(Setting("bears", "SpaceConsistencyTestBear"))
+        self.section1.append(Setting("bears", "SpaceConsistencyTestBear"))
```

Comment:
can you put the same bear in section2 as well and simulate console inputs True, False?